spec.html

```diff
@@ -31728,6 +31744,21 @@ <h1>Properties of the Array Prototype Object</h1>
         <p>The Array prototype object is specified to be an Array exotic object to ensure compatibility with ECMAScript code that was created prior to the ECMAScript 2015 specification.</p>
       </emu-note>
 
+      <emu-clause id="sec-array.prototype.at">
+        <h1>Array.prototype.at ( _index_ )</h1>
+        <emu-alg>
+          1. Let _O_ be ? ToObject(*this* value).
+          1. Let _len_ be ? LengthOfArrayLike(_O_).
+          1. Let _relativeIndex_ be ? ToInteger(_index_).
```

Comment:
Is this `ToIntegerOrInfinity` these days?

Reply:
```suggestion
          1. Let _relativeIndex_ be ? ToIntegerOrInfinity(_index_).
```